ts/private/ts_proto_library.bzl

```diff
@@ -131,6 +131,9 @@ def _protoc_action(ctx, proto_info, outputs):
         ),
         env = {"BAZEL_BINDIR": ctx.bin_dir.path},
```

Comment:
hmm can you check with `-s` what gets sent here? I'm not convinced this `.path` won't break it

Reply:
Does that mean this PR has essentially no effect?

Reply:
yes, i believe so